lib/postRun.js

```diff
@@ -0,0 +1,14 @@
+const { collection } = require("./database");
+
+async function postRun(runName, distance, startDate, pace) {
+  const newrun = await collection("rundata").insertOne({
+    runName,
+    distance,
+    startDate,
+    pace,
+  });
```

Comment:
I'm not too sure, but isn't "insetOne" used for inserting just one argument? Maybe you need to change it to just ".insert"

Reply:
Issue is solved. 👍